data/de/rnv-motis.json

```diff
@@ -0,0 +1,272 @@
+{
+  "name": "Rhein-Neckar-Verkehr (directions.nwex.de)",
+  "attribution": {
+    "license": "https://www.govdata.de/dl-de/by-2-0",
+    "name": "Rhein-Neckar-Verkehr GmbH",
+    "homepage": "https://www.opendata-oepnv.de/ht/de/organisation/verkehrsunternehmen/rnv/openrnv/datensaetze"
```

Comment:
I'm not sure if this is a great value for homepage either, but its also required by the license technically